remotes/push.go

```diff
@@ -235,3 +245,75 @@ func pushBundleConfigDescriptor(ctx context.Context, name string, resolver remot
 	}
 	return descriptor, nil
 }
+
+func pushUnderTag(ctx context.Context, imageClient client.ImageAPIClient, image *bundle.BaseImage, targetRef reference.Named) error {
+	ref := image.Digest
+	if ref == "" {
+		ref = image.Image
+	}
+
+	if err := imageClient.ImageTag(ctx, ref, targetRef.String()); err != nil {
+		return err
+	}
+
+	err := pushTaggedImage(ctx, imageClient, targetRef)
```

Comment:
nit: `return pushTaggedImage(ctx, imageClient, targetRef)`

Reply:
Wonder how the linter didn't catch this.

Reply:
Well, maybe the linter is not well configured, that's my fault I never took time to properly setup it 😅